v0.3/handbook/libraries/stdlib.md

```diff
@@ -1,26 +1,30 @@
 ---
-layout: default
```

Comment:
I don't think you can remove this.... not entirely sure though. It tells Jekyll which layout to use.